### PR TITLE
bluetooth: samples: grammatical fixes

### PR DESCRIPTION
**Pull request by baaatli**
Fixed some of the grammatical errors and spelling mistakes in the README.rst file.

### CODE REVIEW

**Comment by kartben** [comment]
@baaatli is there a reason for closing/reopening new PRs? Anything we can do to help?

**Comment by baaatli** [comment]
Hi @kartben 

Sorry, noobie Zephyrer here :)

I am learning to contribute on Zephyr. I missed some of the checks in the last PR I raised. I did not realise the things like ` before the reference links and hence closed the previous PRs to not waste your time reviewing it. I'll make sure to review them better before raising a PR the next time. Apologies for the inconvenience.

**Comment by kartben** [comment]
No worries:)
You can just keep your PR in draft mode until ira "green" and ready to be reviewed. You can force push amended commit to tour branch, no need to close/reopen all the time :)

**Comment by dipakgmx** [comment]
Hi @baaatli, could you please squash your commits.

**Comment by baaatli** [comment]
@dipakgmx done!

( some mishap in between, sorry :) )

File: samples/bluetooth/channel_sounding/README.rst [hunk 1]
```diff
@@ -7,9 +7,9 @@
 Overview
 ********
 
-These samples demonstrates how to use the Bluetooth Channel Sounding feature.
+These samples demonstrate how to use the Bluetooth Channel Sounding feature.
 
-The CS Test sample shows how to us the CS test command to override randomization of certain channel
+The CS Test sample shows how to use the CS test command to override the  randomization of certain channel
```
Comment (jhedberg) [inline samples/bluetooth/channel_sounding/README.rst:12]: super minor: you have two spaces between "the" and "randomization"

**Review by jhedberg — COMMENTED** [review]
Looks generally fine to me - just a small whitespace issue I commented on separately inline

**Comment by jhedberg** [comment]
Please squash the two commits into a single one

**Review by kartben — DISMISSED** [review]

File: west.yml [hunk 2]
```diff
@@ -199,7 +199,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 0ea9c65e8e14fe01c055e571ad2e6fe5337dc58d
+      revision: d291bdcc4a59bace5ae7453e777e06080ccda8ce
```
Comment (jhedberg) [inline west.yml:202]: What's this? Doesn't seem related to any language fixes.

Comment (baaatli) [reply west.yml:202]: idk where this stemmed from. I didn't change anything in this file.

I did this to squash commits. Maybe this squashed a different commit.

`git fetch
git reset --soft HEAD~2
git commit
git push -f`

Comment (baaatli) [reply west.yml:202]: I think this is too much effort for you folks to have so many reviews for such a small change. I will close this PR, raise a new one in draft mode and make sure it passes the pipeline before raising it for your review. Thanks for your time.

Comment (dipakgmx) [reply west.yml:202]: You can drop that commit and then the PR is good.

Comment (jhedberg) [reply west.yml:202]: @baaatli when you have just a single commit, you can update it simply using `git commit --amend`. You can always check the content what you're about to push using e.g. `git show` (for just a single commit) or e.g. `git log -p upstream/main..`. Also, no need to close this PR and create a new one. Just fix up your local tree, and once you've verified that everything is fine do another `git push --force`.
